tests/checkpoint/test_checkpoint_config.py

```diff
@@ -62,50 +62,19 @@ def checkpoint(
 def test_checkpoint_config_repr(checkpoint):
     checkpoint_config_repr: str = str(checkpoint)
 
-    assert (
-        checkpoint_config_repr
-        == """{
-  "action_list": [
-    {
-      "name": "store_validation_result",
-      "action": {
-        "class_name": "StoreValidationResultAction"
-      }
-    },
-    {
-      "name": "store_evaluation_params",
-      "action": {
-        "class_name": "StoreEvaluationParametersAction"
```

Comment:
The repr has differing outputs due to the variability of the `CheckpointValidationConfig`'s ordering - I figured it was easier to just test some top level keys but lmk if you disagree.